### PR TITLE
docs: Removed and redirected how-to-install-nodejs to downloads page …

### PR DESCRIPTION
**Pull request by Bgregg93**
…(#7184)

## Description

Removed directory and associated files for "how-to-install-NodeJS" and changed URL redirectory to the initial download page (https://nodejs.org/en/download/package-manager)

## Validation
ran multiple checks on the URL

## Related Issues
https://github.com/nodejs/nodejs.org/issues/7184

### Check List

- [ x] I have read the [Contributing Guidelines](https://github.com/nodejs/nodejs.org/blob/main/CONTRIBUTING.md) and made commit messages that follow the guideline.
- [ x] I have run `npm run format` to ensure the code follows the style guide.
- [ x] I have run `npm run test` to check if all tests are passing.
- [x ] I have run `npx turbo build` to check if the website builds without errors.
- [ ] I've covered new added functionality with unit tests if necessary.

### CODE REVIEW

File: apps/site/navigation.json [hunk 1]
```diff
@@ -139,10 +139,6 @@
               "link": "/learn/getting-started/introduction-to-nodejs",
               "label": "components.navigation.learn.gettingStarted.links.introductionToNodejs"
             },
-            "howToInstallNodejs": {
-              "link": "/learn/getting-started/how-to-install-nodejs",
-              "label": "components.navigation.learn.gettingStarted.links.howToInstallNodejs"
```
Comment (AugustinMauroy) [inline apps/site/navigation.json:144]: You have to also remove the i18n key. It's on packages/i18n

**Review by AugustinMauroy — APPROVED** [review]
LGTM!

File: apps/site/navigation.json (hunk 1 above)
Comment (Bgregg93) [reply apps/site/navigation.json:144]: removed the keys in the latest commit sorry for not noticing that.

**Review by benhalverson — APPROVED** [review]
Rad LGTM

**Review by aymen94 — APPROVED** [review]

**Review by bmuenzenmeyer — CHANGES_REQUESTED** [review]
Only removing this content without adding a link in the first page seems to miss the spirit of #7184 

> Maybe we should just add a link to download at the bottom of the ‘introduction-to-nodejs’ page.

It also seems perhaps premature to remove this without thinking holistically about #7198 - it did predate our larger effort by 2 days.

**Comment by Bgregg93** [comment]
> Only removing this content without adding a link in the first page seems to miss the spirit of nodejs/nodejs.org#7184

yes sorry this was an oversight, added the link to the bottom, was not sure where it should of been placed or what specific format was wanted.

File: apps/site/pages/en/learn/getting-started/introduction-to-nodejs.md [hunk 2]
```diff
@@ -96,3 +96,5 @@ and we close the response, adding the content as an argument to `end()`:
 ```js
 res.end('Hello World\n');
 ```
+
+[Download node.js](https://nodejs.org/en/download/package-manager)
```
Comment (AugustinMauroy) [inline apps/site/pages/en/learn/getting-started/introduction-to-nodejs.md]:
```suggestion
If you haven't already done so, [download](https://nodejs.org/en/download) Node.js.
```

Comment (bmuenzenmeyer) [reply apps/site/pages/en/learn/getting-started/introduction-to-nodejs.md]: isn't Node.js always supposed to be capitalized when in writing ?

**Review by bmuenzenmeyer — APPROVED** [review]

**Pull request merged**